vulcanai2/tests/models/test_layers.py

```diff
@@ -82,22 +82,24 @@ def conv_unit(self):
     def test_dense_selu_weight_change(self, dense_unit):
         """Confirm SELU weight init properties hold for dense net."""
         starting_weight = copy.deepcopy(dense_unit._kernel.weight)
-        std = round(math.sqrt(1. / 10), 1)
+        fan_in = dense_unit.in_features
+        std = round(math.sqrt(1. / fan_in), 1)
 
         dense_unit.weight_init = selu_weight_init_
         dense_unit._init_weights()
-        new_weight = copy.deepcopy(dense_unit._kernel.weight)
+        new_weight = dense_unit._kernel.weight
         assert (torch.equal(starting_weight, new_weight) is False)
         assert pytest.approx(round(new_weight.std().item(), 1), 0.1) == std
         assert (int(new_weight.mean().item()) == 0.0)
 
     def test_conv_selu_weight_change(self, conv_unit):
         """Confirm SELU weight init properties hold for conv net."""
         starting_weight = copy.deepcopy(conv_unit._kernel.weight)
-        std = round(math.sqrt(1. / 250), 1)
+        fan_in = conv_unit._kernel.in_channels * conv_unit._kernel.kernel_size[0] * conv_unit._kernel.kernel_size[1]
```

Comment:
you can probably use the fan_in pytorch functionality?

Reply:
or you could make it independent of the shape of the kernel (i.e. right now it will only work with 2D convo kernels)

Reply:
@rfratila Made the update to be able to handle different sized kernels